src/backend_model.cc

```diff
@@ -114,42 +114,58 @@ TritonModel::Create(
       JoinPath({localized_model_path, std::to_string(version)});
   const std::string global_path =
       JoinPath({backend_dir, specialized_backend_name});
-  const std::vector<std::string> search_paths = {
+  std::vector<std::string> search_paths = {
       version_path, localized_model_path, global_path};
 
   std::string backend_libdir;
   std::string backend_libpath;
-  for (const auto& path : search_paths) {
-    const auto full_path = JoinPath({path, backend_libname});
+  RETURN_IF_ERROR(LocateBackendLibrary(
```

Comment:
Question: could LocateBackendLibaray try to find model.py as well as .so / .dll and return a bool flag for a 'python_backend" - that way the search paths for .so and model.py would be the same?